Makefile

```diff
@@ -7,7 +7,6 @@ TEST_OS = fedora-32
 endif
 export TEST_OS
 TARFILE=cockpit-$(PACKAGE_NAME)-$(VERSION).tar.gz
-RPMFILE=$(shell rpmspec -D"VERSION $(VERSION)" -q cockpit-podman.spec.in).rpm
```

Comment:
How about supporting both .spec.in and .spec here? something like

    ... -q `ls cockpit-podman.spec.in cockpit-podman.spec 2>/dev/null| head -n1`).rpm

?

Reply:
yeah, that was my other idea to just check which one is present. Found it a bit hacky to do that, but if we want to keep the current behaviour, then I can go with that. Will just have to check if rpmspec with specified version behaves the same for spec.in and spec.